composer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "alleyinteractive/options-importer",
+  "type": "wordpress-plugin",
+  "keywords": ["wordpress", "plugin"],
+  "authors": [
+    {
+      "name": "Alley Interactive",
+      "email": "noreply@alleyinteractive.com"
+    }
+  ],
+  "require": {
+    "composer/installers": "~1.0"
+  },
+  "require-dev": {
+    "squizlabs/php_codesniffer": "3.*",
+    "wp-coding-standards/wpcs": "2.*"
```

Comment:
Could VIPCS be tested as well?

If so, then I'd also recommend using the https://github.com/Dealerdirect/phpcodesniffer-composer-installer/ package which will register all of the sniff packages automatically.